app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
it is a bad code style to have brackets at different levels

While creating a dictionary/list - the braces must be located in one of two options: open and start with the text or have a line break between the text.